requirements/base.in

```diff
@@ -2,46 +2,49 @@
 # like Pillow and psycopg2
 # See http://bitly.com/wheel-building-fails-CPython-35
 # Verified bug on Python 3.5.1
-wheel==0.29.0
+wheel
 
-django==1.9.4
+django
```

Comment:
Won't not specifying versions in the `.in` file cause them to always be upgraded in the `.txt` file to whatever happens to be the latest? This sounds a bit dangerous. AFAIK the point of the `.in` file is to specify direct dependencies (but still lock them) and the full dependencies then get pulled into the `.txt` file.

I guess if we can be sure that we don't care what version of any of the packages we require on are, as long as they are compatible. But somehow I think that is going to be an idealistic case, with many packages shipping breaking changes in even minor updates.